lib/reek/cli/command/todo_list_command.rb

```diff
@@ -36,13 +36,13 @@ def scan_for_smells
         end
 
         def groups_for(smells)
-          @groups ||= begin
-            Hash[
-              smells.group_by(&:smell_type).map do |smell_type, smells_for_type|
-                [smell_type, { 'exclude' => smells_for_type.map(&:context) }]
+          @groups ||=
+            begin
+              todos = smells.group_by(&:smell_class).map do |smell_class, smells_for_class|
```

Comment:
Sorry for being ambiguous. And actually I'd rather retract my statement. Why not stick with `SmellWarning#smell_type` but instead using another method? If you'd go ahead with that change we'd have `SmellWarning#smell_type`  and `SmellWarning#smell_class` and both would return the same.